src/UserAdmin.php

```diff
@@ -68,10 +66,10 @@ public function setModel(Model $user)
 
                 return [
                     $v->owner->hide(),
-                    $this->notify = new \atk4\ui\jsNotify([
+                    $this->notify = new jsNotify([
```

Comment:
@DarkSide666 - Do we want to use jsToast here instead? jsNotify might not be supported eventually.

Reply:
done